### PR TITLE
Move detail progress monitor to run_dialog

### LINKED ISSUE #280: Let the user use progress window to debug forward models
It should be possible to do basic debugging of your forward model using the progress window. To achieve this one needs to support:
  - #274 (displaying the error messages dumped to file),
  - as well as keeping the data available also after the end of an ensemble has finished (possibly with all the realizations failing).

### PR DESCRIPTION
**Pull request by ManInFez**
Add support investigating past iterations

**Task**
Upgrade realizations view after simulation finished to the forward model monitoring widget. 
Add support to show multiple iterations in the monitoring widget
Allow users to debug if the run fails completely

Resolves #280

### CODE REVIEW

File: python/python/ert_gui/simulation/detailed_progress.py [hunk 1]
```diff
@@ -257,27 +266,23 @@ def reload(self, file_name):
         self.show()
 
 
-class DetailedProgressDialog(QDialog):
+class DetailedProgressDialog(QWidget):
```
Comment (andreabrambilla) [inline python/python/ert_gui/simulation/detailed_progress.py]: Well, we could also consider changing the class name ...

File: python/python/ert_gui/simulation/run_dialog.py [hunk 2]
```diff
@@ -20,60 +39,52 @@
 class RunDialog(QDialog):
 
     def __init__(self, run_model, parent):
-        QDialog.__init__(self, parent)
+        QDialog.__init__(self, None)
```
Comment (andreabrambilla) [inline python/python/ert_gui/simulation/run_dialog.py]: Why did you change this?

Comment (ManInFez) [reply python/python/ert_gui/simulation/run_dialog.py]: The parent window flags caused strange window behaviour. Changing it to default window flag in and setting parent in next commit.

File: python/python/ert_gui/simulation/detailed_progress.py (hunk 1 above)
Comment (ManInFez) [reply python/python/ert_gui/simulation/detailed_progress.py]: Yes we should :)
